API/Osu/Multiplayer/OsuApiService.cs

```diff
@@ -19,6 +19,9 @@ public class OsuApiUser
 
     [JsonProperty("country")]
     public string Country { get; set; } = null!;
+
+    [JsonProperty("ruleset")]
```

Comment:
I think we can actually remove all of the JsonProperty fields in this class, I don't see them being used anywhere.

Reply:
I was wondering about those, didn't understand what they were used for so I wasn't going to touch them lol